Packs/GoogleDrive/Integrations/GoogleDrive/README.md

```diff
@@ -1,88 +1,30 @@
-Google Drive allows users to store files on their servers, synchronize files across devices, and share files. This integration helps you to create a new drive, query past activity and view change logs performed by the users. 
-This integration was integrated and tested majorly with G Suite Basic and Enterprise edition.
+Google Drive allows users to store files on their servers, synchronize files across devices, and share files. This integration helps you to create a new drive, query past activity, and view change logs performed by the users.
+This integration was integrated and tested with version xx of GoogleDrive
```

Comment:
remove this line

Reply:
https://github.com/demisto/content/pull/13643/commits/a0e95a499d78e6acb52adc2d7faf7af13834a0fb